SQL/SQL.sublime-syntax

```diff
@@ -243,3 +249,31 @@ contexts:
       # Handle situations where the schema and . 
     - match: '{{end_identifier}}'
       pop: true
+  like_escape:
+    - match: (\')([^'])(\')
+      captures:
+        0: meta.string.escape.sql string.quoted.single.sql
+        1: punctuation.definition.string.begin.sql
+        2: constant.character.escape.sql
+        3: punctuation.definition.string.end.sql
+      pop: true
+    - match: (?=\S)
+      pop: true
+  like_string:
+    - match: \'
+      scope: punctuation.definition.string.begin.sql
+      set: inside_like_single_quoted_string
+    - match: (?=\S)
+      pop: true
+  inside_like_single_quoted_string:
+    - meta_scope: meta.string.like.sql string.quoted.single.sql
+    - match: \'
+      scope: punctuation.definition.string.end.sql
+      pop: true
+    - match: (\[)(?:.|[^]']+?)(\])
+      scope: meta.set.like.sql
+      captures:
+        1: keyword.control.set.begin.sql
+        2: keyword.control.set.end.sql
+    - match: '[%_]'
+      scope: keyword.other.any.sql
```

Comment:
Looks like `%` and `_` are wildcards just like `*` and `?` in path patterns?

see: https://docs.microsoft.com/en-us/sql/t-sql/language-elements/like-transact-sql?view=sql-server-ver15

Maybe one of the following scopes would be appropriate then:
- `variable.language.wildcard`
- `keyword.operator.wildcard`

Both are used in several syntaxes, already. Not sure which one would be the better choise, maybe the latter?

Reply:
FWIW, I took `keyword.other.any` from https://github.com/sublimehq/Packages/blob/cd881081814883bdceecdcb8bdc2343cc46bc639/Regular%20Expressions/RegExp.sublime-syntax#L352 - maybe it would also make sense to standardize this one to `keyword.operator.wildcard`.